### ISSUE TITLE
Fish Alias not working.

### ISSUE THREAD

**Issue by LinArcX** [ranger/ranger#1413]
#### Runtime Environment

- Operating system and version: Arch 
- Terminal emulator and version: konsole 18.12.0
- Python version: 3.7.1 (default, Oct 22 2018, 10:41:28) [GCC 8.2.1 20180831
- Ranger version/commit: ranger 1.9.2
- Locale: en_US.UTF-8

#### Current Behavior
I want to bind some fish aliases to shortkeys in ranger. but they don't work and show me some errors.

#### Expected Behavior
:)

#### Steps to reproduce
1. Install fish and set it as defualt shell
2. create some aliases and put them in __~/.config/fish/config.fish__.
3. use those aliases to bind for your favorite keybindings.

#### Traceback

```
/bin/sh: enc: command not found
Error detected while processing function <SNR>114_PollServerReady[1]..<SNR>114_Pyeval[2]..provider#python3#Call:
line   18:
Invalid channel: 4
Error detected while processing function <SNR>114_PollServerReady[7]..<SNR>114_Pyeval[2]..provider#python3#Call:

```

**Comment by Vifon** [comment]
Do you have `if status is-login ...` in your config.fish? Usually in such cases I recommend turning these aliases into external scripts, one shouldn't expect the aliases to work outside of an interactive shell.

**Comment by toonn** [comment]
I agree, aliases don't really have a place in scripts and the ranger config is sufficiently like a script.
